pkg/plugins/runtime/k8s/webhooks/validation.go

```diff
@@ -72,6 +102,28 @@ func (h *validatingHandler) Handle(ctx context.Context, req admission.Request) a
 	return admission.Allowed("")
 }
 
+func (h *validatingHandler) validateSync(resType core_model.ResourceType, obj k8s_model.KubernetesObject) admission.Response {
+	if resType == core_mesh.MeshType && len(obj.GetSpec()) == 0 { // skip validation for the default mesh
```

Comment:
The comment says that we skip default mesh, but it skips every mesh with empty spec, isn't it? And why do we have to skip default mesh? Is it created by `defaultingWebhook`?

Reply:
Because Remote can create default Mesh and if we let the logic pass, it will fail because normally you should not create meshes on Remotes.

Reply:
I don't know... mesh defaulter does not make sense on remote 🤔  it's only useful when you apply a mesh with some settings, but it will be defaulted in global